CHANGELOG.md

```diff
@@ -38,6 +38,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added: Use Speedbooster to skip breaking the blob submerged in water in Artaria Early Cloak room, requires SpeedBooster Conservation (Beginner).
 - Added: Use Flash Shift to go right after getting the pickup in Artaria EMMI Zone Spinner.
 - Added: Use Flash Shift and Slide Jump to go from Artaria White EMMMI Arena to the top door to EMMI Zone Spinner.
+- Added: Use a Shinespark to climb up from Above Screw Attack Blocks in Burenia Main Hub Tower Bottom with only Gravity Suit.
+- Added: Use a Shinespark to climb up from Alcove Across Grapple Block in Burenia Main Hub Tower Bottom with only Speedbooster using Speedbooster Conservation Beginner.
+- Added: Use a Shinespark with Gravity Suit to reach Ammo Recharge South at the bottom of Burenia Gravity Suit Tower before the Destroy Gravity Suit Floor event.
+- Added: Use Spin Boost And Gravity Suit with different trick strategies to cross the big gap in Burenia Main Hub Tower Middle.
+- Added: Use a Shinespark with Gravity Suit to reach the Spider Magnet wall in Burenia Main Hub Tower Middle from the bottom of the room.
+- Added: Climb up to the Charge Beam Door in Burenia Main Hub Tower Middle using Gravity Suit and either Spin Boost or Flash Shift.
```

Comment:
This is already a thing, connected from the lower tunnel to that room. I'm not sure how Flash can get to it though.

Reply:
Ok, I'll move the flash shift to that connection then. The way it works is that you can simply walljump and then flash onto the ledge 
![image](https://user-images.githubusercontent.com/6470224/232213887-af7ab058-1cf8-4046-90ba-663ccf90d5aa.png)